fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1396,4 +1396,12 @@
      */
     @ConfField(mutable = true, masterOnly = true)
     public static int max_dynamic_partition_num = 500;
+
+    /**
+     * min/max replica number limit
+     */
+    @ConfField(mutable = true)
```

Comment:
The minimum and maximum values should not be a configuration, but a hard-coded constant.